phpswitch.sh

```diff
@@ -2,22 +2,41 @@
 # Creator: Phil Cook
 # Email: phil@phil-cook.com
 # Twitter: @p_cook
+brew_prefix=$(brew --prefix | sed 's#/#\\\/#g')
+
 php_array=("php53" "php54" "php55" "php56")
 php_installed_array=()
 php_version="php$1"
-apache_conf_path="/etc/apache2/httpd.conf"
-brew_prefix=$(brew --prefix | sed 's#/#\\\/#g')
 php_opt_path="$brew_prefix\/opt\/"
-php_lib_path="\/libexec\/apache2\/libphp5.so"
-php_mod_path="$php_opt_path$php_version$php_lib_path"
+
+apache_change=1
+apache_conf_path="/etc/apache2/httpd.conf"
+apache_php_lib_path="\/libexec\/apache2\/libphp5.so"
+apache_php_mod_path="$php_opt_path$php_version$apache_php_lib_path"
 
 # Has the user submitted a version required
 if [[ -z "$1" ]]
 then
-	echo "Please set which version you want to switch to. E.g. brew-php-switcher 53"
+	echo "usage: brew-php-switcher version [-s]"; echo;
+	echo "    version    one of:" ${php_array[@]};
```

Comment:
Again this needs to be the 53,54,55,56 format and not the php53,php54,php55,php56 format.